content/reproducibility.md

```diff
@@ -18,11 +18,11 @@ keywords:
 Science is hard, why make it harder?
 ------------------------------------
 
-Scientists and researchers spend a lot of time on data preparation and analysis, and some of these analyses are quite computationally intensive. The amount of time required to conduct an analysis grows for the increased complexity of calculations, amount of data, and number of datasets that will be analyzed. Many researchers use spreadsheets to conduct their analysis workflows, and apply the same set to each data set manually. A majority of the time in this type of workflow could be spent copying and pasting equations from one spreadsheet or column to another. This spreadsheet workflow is highly error-prone, as *The Economist* shows in "[Excel errors and science papers](http://www.economist.com/blogs/graphicdetail/2016/09/daily-chart-3?fsrc=scn/fb/te/bl/ed/excelerrorsandsciencepapers)".
+Scientists and researchers spend a lot of time on data preparation and analysis, and some of these analyses are quite computationally intensive. The amount of time required to conduct an analysis grows for the increased complexity of calculations, amount of data, and number of datasets that will be analyzed. Many researchers use spreadsheets to conduct their analysis workflows, and apply the same tasks to each data set manually. A majority of the time in this type of workflow could be spent copying and pasting equations from one spreadsheet or column to another.
```

Comment:
Maybe time and effort instead of computationally, and maybe also specify to manually/without automation?  As you're arguing, computationally intensive doesn't have to equal time intensive :)